website/src/css/custom.css

```
@@ -445,3 +445,48 @@ ul.circle > li:before {
   }
 } 
 
+.feedback-widget-container {
+  position: fixed;
+  right: 0;
+  border: 1px solid #ccc;
+  border-radius: 8px;
+  padding: 0.5em !important;
+  text-align: center;
+  width: 17%;
+  max-height: 25%;
+  margin-top: 2em !important;
+  margin-right: 1.5em !important;
+
+}
```

removed `position: fixed` and set `width: 100%`

```suggestion
.feedback-widget-container {
  border: 1px solid #ccc;
  border-radius: 8px;
  padding: 0.5em !important;
  text-align: center;
  width: 100%;
  margin-top: 2em !important;
  margin-right: 1.5em !important;
}
```

---

position: fixed is needed so that the feedback widget is always available even if the page is scrolling. With position, width is also needed.